src/components/Feed/Feed.js

```diff
@@ -3,12 +3,38 @@ import { Route } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import './Feed.css';
+import Post from '../../components/Post/Post';
+import { user, posts } from '../../Data/mockData';
 
-// import Post from '../../components/Post/Post';
+const Feed = ({ profileIsVisible }) => {
+  const [currentUserPosts, setCurrentUserPosts] = useState([])
+  const [allPosts, setAllPosts] = useState([])
+
+  useEffect(() => {
+    setCurrentUserPosts(user.posts)
+    setAllPosts(posts)
+  }, [currentUserPosts, allPosts])
```

Comment:
Excited to see this working - glad you figured out that issue for the posts to be added!